reconcile/openshift_resources.py

```diff
@@ -172,6 +177,22 @@ def fetch_provider_vault_secret(path, version, name, labels, annotations):
     return openshift_resource
 
 
+def fetch_provider_route(path, tls_path, tls_version):
+    openshift_resource = fetch_provider_resource(path)
+
+    if tls_path is None or tls_version is None:
```

Comment:
can we log this, so this can be tracked? let's say someone specifies tls_path but not tls_version, then the tls section would not be replaced, and nothing would fail. I'd like some logging for this

Reply:
as discussed, this will be handled as a schema validation